docs/config/README.md

```diff
@@ -181,6 +181,7 @@ format = "$all"
 format = """
 $username\
 $hostname\
+$localipv4\
```

Comment:
This needs to be reverted, I think. Otherwise the module doesn't show up for me at all.
```suggestion
$localip\
```